### PR TITLE
Implement Graphite Carbon Api

### PR DESCRIPTION
**Pull request by qwertBR**
### Changelog category:
- New Feature

### CODE REVIEW

File: src/Server/GraphiteFinder.cpp [hunk 1]
```diff
@@ -0,0 +1,273 @@
+
```
Comment (evillique) [inline src/Server/GraphiteFinder.cpp:1]: Code should be formatted in accordance with our [code style](https://clickhouse.com/docs/en/development/style/).  Most of the work could be done with `clang-format`.

File: programs/server/uuid [hunk 2]
```diff
@@ -0,0 +1 @@
+f3f739e0-99d2-4e4e-83a3-7e4a7d349b67
```
Comment (evillique) [inline programs/server/uuid:1]: This file was probably added by mistake

File: src/Server/GraphiteFinder.cpp [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+#include "GraphiteUtils.h"
+#include "fmt/format.h"
+#include <regex>
+#include <chrono>
+
+namespace DB
+{
+
+
+int IndexDirect = 1;
```
Comment (evillique) [inline src/Server/GraphiteFinder.cpp:13]: Having global non-const variables in the DB namespace is probably not the best idea.

**Review by evillique — COMMENTED** [review]
Could you please fix formatting, add a test and update your branch to the latest master?

File: src/Server/GraphiteUtils.cpp [hunk 4]
```diff
@@ -0,0 +1,586 @@
...
+#include <vector>
+#include <fmt/format.h>
+
+namespace GraphiteCarbon
+{
+
+
+const std::string op_eq = "=";
+
+std::string join(const std::vector<std::string> & elems, size_t start, size_t end, const std::string & sep)
```
Comment (alexey-milovidov) [inline src/Server/GraphiteUtils.cpp:18]: The code is strange.

**Comment by alexey-milovidov** [comment]
Unfinished.

**Pull request closed without merging**